domain/user/errors/errors.go

```diff
@@ -8,9 +8,9 @@ import (
 )
 
 const (
-	// NotFound describes an error that occurs when the user being requested does
+	// UserNotFound describes an error that occurs when the user being requested does
```

Comment:
No need to prefix these. This is implied by the package. The other domain error packages do not do this.

Reply:
Ok, maybe I incorrectly understand that we last decided to be more explicit in error definitions.